Source/WebKit/Shared/Cocoa/CoreIPCData.serialization.in

```diff
@@ -24,7 +24,7 @@
 
 webkit_platform_headers: "CoreIPCData.h"
 
-[WebKitPlatform] class WebKit::CoreIPCData {
+[WebKitPlatform, AdditionalEncoder=StreamConnectionEncoder] class WebKit::CoreIPCData {
     IPC::DataReference get();
```

Comment:
I needed to add: [NotSerialized](https://github.com/WebKit/WebKit/compare/491a015b27ed49fef99abb6fe466dd71f54f1553..79da2a56ae4a85c4901d09d6e0b00908f4fce05f#diff-c53573c4702dac302bb681f594eb635030e08018baf28929fd51e790959d355fL26) before, as I hit a static assert. Do we want that same check?

Reply:
My evidence suggests we do not need that, because I'm running tests just fine with no static assert.

Reply:
That said, I'm staring *right at* the generated serializers here...

`void ArgumentCoder<WebKit::CoreIPCData>::encode(Encoder& encoder, const WebKit::CoreIPCData& instance)
{
    static_assert(std::is_same_v<std::remove_cvref_t<decltype(instance.get())>, IPC::DataReference>);

    encoder << instance.get();
}

void ArgumentCoder<WebKit::CoreIPCData>::encode(StreamConnectionEncoder& encoder, const WebKit::CoreIPCData& instance)
{
    static_assert(std::is_same_v<std::remove_cvref_t<decltype(instance.get())>, IPC::DataReference>);

    encoder << instance.get();
}`

And I'm not sure how that's *not* firing

Reply:
Ohhhh  that's typing, not size.

So yah, something about this declaration is not putting the size in, so...  *shrug*